### PR TITLE
Events in v1.5: added `node_relation` to `node_info`

### LINKED ISSUE #3167: In events/logs, `node_info` includes nested `node_relation` dict
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

https://docs.getdbt.com/reference/events-logging#node_info-fields

### What part(s) of the page would you like to see updated?

Add `node_relation`, which is a dictionary including:
- `database`
- `schema`
- `alias`
- `relation_name` (full stringified value with include/quoting rules applied)

### Additional information

Relevant PR:
- https://github.com/dbt-labs/dbt-core/pull/7304

### PR DESCRIPTION
**Pull request by jtcohen6**
resolves #3167

## What are you changing in this pull request and why?

In v1.5, structured events will include a `node_relation` object, nested within the `node_info` object, for node-specific events

## Checklist
- [x] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
- [x] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

### CODE REVIEW

File: website/docs/guides/migration/versions/02-upgrading-to-v1.5.md [hunk 1]
```diff
@@ -77,3 +77,6 @@ More to come!
 
 ### dbt-core Python API
 - Auto-generated documentation ([#2674](https://github.com/dbt-labs/docs.getdbt.com/issues/2674)) for dbt-core CLI & Python API for programmatic invocations
+
+### Quick hits
```
Comment (jtcohen6) [inline website/docs/guides/migration/versions/02-upgrading-to-v1.5.md:81]: This is a silly name for the section header — but I figure I'll rename it, and add in more of the small stuff, during v1.5 guide revision (#3170)

File: website/docs/reference/events-logging.md [hunk 2]
```diff
@@ -65,6 +67,7 @@ For events where it's available, `node_info` will include:
 | `node_finished_at` | Timestamp when node processing completed |
 | `node_name` | Name of this model/seed/test/etc |
 | `node_path` | File path to where this resource is defined |
+| `node_relation` | <VersionBlock firstVersion="1.5">Nested object containing this node's database representation: `database`, `schema`, `alias`, and full `relation_name` with quoting & inclusion policies applied</VersionBlock><VersionBlock lastVersion="1.4">Added in [v1.5](/reference/events-logging?version=1.5#node_info-fields)</VersionBlock> |
```
Comment (jtcohen6) [inline website/docs/reference/events-logging.md]: This is me being a little bit fancy with `VersionBlock`

Screenshot 2023-04-12 at 21 45 07

Screenshot 2023-04-12 at 21 44 57

Comment (dbeatty10) [reply website/docs/reference/events-logging.md]: 🤩

FFFTW! (Fancy Footwork For The Win)

File: website/docs/guides/migration/versions/02-upgrading-to-v1.5.md [hunk 3]
```diff
@@ -77,3 +77,6 @@ More to come!
...
+- [Events and logging](events-logging): Added `node_relation` (`database`, `schema`, `identifier`) to the `node_info` dictionary, available on node-specific event
```
Comment (dbeatty10) [inline website/docs/guides/migration/versions/02-upgrading-to-v1.5.md]: Should "event" be pluralized?

```suggestion
- [Events and logging](events-logging): Added `node_relation` (`database`, `schema`, `identifier`) to the `node_info` dictionary, available on node-specific events
```

**Review by dbeatty10 — DISMISSED** [review]

File: website/docs/guides/migration/versions/02-upgrading-to-v1.5.md (hunk 3 above)
Comment (jtcohen6) [reply website/docs/guides/migration/versions/02-upgrading-to-v1.5.md]: sure should!

File: website/docs/reference/events-logging.md (hunk 2 above)
Comment (matthewshaver) [reply website/docs/reference/events-logging.md]: This is a fantastic idea, but it doesn't function as expected due to technical limitations. We've tried this in the past, and the behavior here is the same - it links to the page properly but doesn't change the version of the docs when written as a markdown link. It works if you put the full `HTTP` link, but the downside there is it opens in a new tab (this is expected behavior in the current Docusaurus implementation).

Comment (jtcohen6) [reply website/docs/reference/events-logging.md]: Ah - alas. I'll remove the link, but leave in the conditional table contents

**Review by dbeatty10 — DISMISSED** [review]

**Review by matthewshaver — DISMISSED** [review]

**Comment by jtcohen6** [comment]
Going to merge this in since it has prior approval from @dbeatty10 and @matthewshaver

**Pull request merged**